Helper/Data.php

```diff
@@ -1733,6 +1733,7 @@ public function createAdyenBillingAgreement($order, $additionalData)
             $comment = $order->addStatusHistoryComment($message);
 
             $order->addRelatedObject($comment);
+            $order->save();
```

Comment:
create ticket to save the order not with the order object but with a resource model